README.md

```diff
@@ -129,19 +129,56 @@ DJANGO_WORKER_COUNT=1 DJANGO_WORKER_INDEX=0SPANNER_TEST_INSTANCE=django-tests ./
 
 ## Limitations
 
-Limitation|Comment|Resolution
----|---|---
-Lack of DEFAULT for columns|Cloud Spanner doesn't support using DEFAULT for columns thus the use of default values might have to enforced in your controller logic|
```

Comment:
We definitely need to bring this back, quite important to let users know that they can't rely on defaults.

Reply:
Django doesn't set database defaults (except temporarily to set values on new columns; django-spanner uses an UPDATE query to behave equivalently).

Reply:
Awesome then, thanks for the update.